src/host/cmdline.h

```diff
@@ -131,18 +131,18 @@ void DeleteCommandLine(COOKED_READ_DATA& cookedReadData, const bool fUpdateField
 void RedrawCommandLine(COOKED_READ_DATA& cookedReadData);
 
 // Values for WriteChars(), WriteCharsLegacy() dwFlags
-#define WC_DESTRUCTIVE_BACKSPACE 0x01
+#define WC_INTERACTIVE 0x01
 #define WC_KEEP_CURSOR_VISIBLE 0x02
-#define WC_PRINTABLE_CONTROL_CHARS 0x04
 
 // This is no longer necessary. The buffer will always be Unicode. We don't need to perform special work to check if we're in a raster font
 // and convert the entire buffer to match (and all insertions).
-//#define WC_FALSIFY_UNICODE       0x08
-
-#define WC_LIMIT_BACKSPACE 0x10
-//#define WC_NONDESTRUCTIVE_TAB    0x20  -  This is not needed anymore, because the VT code handles tabs internally now.
-//#define WC_NEWLINE_SAVE_X        0x40  -  This has been replaced with an output mode flag instead as it's line discipline behavior that may not necessarily be coupled with VT.
-//#define WC_DELAY_EOL_WRAP        0x80  -  This is not needed anymore, because the AdaptDispatch class handles all VT output.
+//#define WC_DESTRUCTIVE_BACKSPACE   0x01  -  Replaced with WC_INTERACTIVE
+//#define WC_PRINTABLE_CONTROL_CHARS 0x04  -  Replaced with WC_INTERACTIVE
+//#define WC_FALSIFY_UNICODE         0x08

```

Comment:
The comment for `WC_FALSIFY_UNICODE` is now in the wrong place.

Reply:
At this point, maybe we should just delete the old WCs completely. They were nice to have in the SD and early Git days when Microsoft changed up the Windows repo every couple months and lost all the history (or made it hard to access), but now we have git blame and stuff and a completely intact repo history.

I do like archaeology though...

Reply:
Yeah, I kinda felt the same way. Especially as I now continue to clean all this up, these flags will soon likely lose all of their remaining meaning.